packages/rocketchat-automatic-channels-handler/server/server.js

```diff
@@ -0,0 +1,77 @@
+export const automaticChannelsHandler ={};
+automaticChannelsHandler.categories = [];
+
+function removeUserFromOtherAutomaticChannels(user, categories) {
+	const channelNames = categories.map(function(x) { return x.getChannelName(user); });
+
+	const userSubscriptions = RocketChat.models.Subscriptions.findByTypeAndUserId('c', user.user._id).fetch();
```

Comment:
Can you check if there was returned channelNames (if plugins are enabled) before get all subscriptions? This is heavy